### PR TITLE
Perceptual speed index

### LINKED ISSUE #19: Perceptual Speed Index
FYI pat meenan now has a standalone project for calculating metrics like this: https://github.com/WPO-Foundation/visualmetrics

It includes perceptual speed index; https://github.com/WPO-Foundation/visualmetrics/blob/master/visualmetrics.py#L820-L855

While SI uses color distribution to compare two frames, perSI uses [SSIM](https://en.wikipedia.org/wiki/Structural_similarity) instead.

What do you think about doing the same here in speedline? [image-ssim](https://github.com/darosh/image-ssim-js) looks very well done. Hmm!

### PR DESCRIPTION
**Pull request by pmdartus**
Implement `Perceptual Speed Index`.
Will fix #19

### CODE REVIEW

**Comment by paulirish** [comment]
This is on the right path.

The numbers are going to be v different but its up to our human brains to determine which of these is better. I think we should land this to get a little more familiar with the results.

**Comment by paulirish** [comment]
We've spent time with the folks at InstartLogic who've spearheaded this metric. As a result, we have even more confidence that pSI is superior to SI. There's a deck I'm going to get you access to.

Can we land this? Even excited to have this reporting come through with a flag.
If not, I'm likely going to fork for now. :)

**Comment by pmdartus** [comment]
I am currently busy because of work. I will finish the PR by Sunday.
Is it good for you ? =)

**Comment by samccone** [comment]
:+1: no rush @pmdartus let me know if I can be of any help.

**Comment by pmdartus** [comment]
@paulirish @samccone Ready for review.
Here are some results comparing the speed index and the perceptual speed index :

| site                              | Speed Index | Perceptual Speed Index |
|-----------------------------------|-------------|------------------------|
| nytimes.com                       | 4148        | 4980                   |
| google.com                        | 602         | 622                    |
| travis.com                        | 1803        | 2645                   |
| events.google.com/io2016/schedule | 1205        | 1334                   |
| youtube.com                       | 2275        | 3732                   |

**Comment by pmdartus** [comment]
The only method

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "babar": "0.0.3",
     "devtools-timeline-model": "^1.1.2",
+    "image-ssim": "^0.2.0",
```
Comment (samccone) [inline package.json]: :heart: 
https://github.com/darosh/image-ssim-js/blob/master/package.json#L11

**Comment by samccone** [comment]
LGTM :shipit:

**Comment by pmdartus** [comment]
@samccone thanks for the review
Landed in `0.2.0` 👍

**Pull request merged**